Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -1016,13 +1017,298 @@ def set_user_command(args):
         return responseJson
 
 
+def create_ip_destination_group(args: dict):
+    headers = ['ID', 'Name', 'Type', 'Description', 'Addresses', 'Countries', 'IpCategories', 'IsNonEditable']
+    payload = {
+        'name': args.get('name', ''),
+        'type': args.get('type', ''),
+        'countries': argToList(args.get('countries', '')),
+        'ipCategories': argToList(args.get('ip_categories', '')),
+        'description': args.get('description', ''),
+        'addresses': argToList(args.get('addresses', '')),
+        'isNonEditable': args.get('is_non_editable', False)
+    }
+    cmd_url = "/ipDestinationGroups"
+    try:
+        response = http_request('POST', cmd_url, data=json.dumps(payload),
+                                headers=DEFAULT_HEADERS).json()
+    except json.decoder.JSONDecodeError as exp:
+        return_error(f'Failed to execute zscaler-create-ip-destination-group command. Error: {str(exp)}')
+    content = {
+        'ID': int(response.get('id', '')),
+        'Name': response.get('name', ''),
+        'Type': response.get('type', ''),
+        'Description': response.get('description', ''),
+        'Addresses': response.get('addresses', []),
+        'IpCategories': response.get('ipCategories', []),
+        'Countries': response.get('countries', []),
+        'IsNonEditable': response.get('isNonEditable', False)
+    }
+    entry = {
+        'Type': entryTypes['note'],
+        'Contents': content,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown("IPv4 Destination group created",
+                                         content, headers, removeNull=True),
+        'EntryContext': {'Zscaler.IPDestinationGroup': content},
+    }
+    return entry
+
+
+def list_ip_destination_groups(args: dict):
```

Comment:
there's a lot of repetative code in this method, we can do something like:
```
lite_endpoint = "/lite" if lite else ""
ipv4_cmd_url = "/ipDestinationGroups" + lite_endpoint
ipv4_responses = http_request('GET', ipv4_cmd_url).json()
ipv4_contents = get_contents_lite(ipv4_responses) if lite else get_contents(ipv4_responses)
...
if include_ipv6:
    ipv6_cmd_url = "/ipDestinationGroups/ipv6DestinationGroups" + lite_endpoint
    ipv6_responses = http_request('GET', ipv6_cmd_url).json()
    ipv6_contents = get_contents_lite(ipv6_responses) if lite else get_contents(ipv6_responses)
    ...
```

Reply:
done